src/components/views/elements/StyledRadioButton.tsx

```diff
@@ -6,11 +6,11 @@ SPDX-License-Identifier: AGPL-3.0-only OR GPL-3.0-only
 Please see LICENSE files in the repository root for full details.
 */
 
-import React from "react";
+import React, { RefCallback } from "react";
 import classnames from "classnames";
 
 interface IProps extends React.InputHTMLAttributes<HTMLInputElement> {
-    inputRef?: React.RefObject<HTMLInputElement>;
+    inputRef?: RefCallback<HTMLInputElement>;
```

Comment:
Could this not be just `Ref` to accept either?

Reply:
https://github.com/element-hq/element-web/pull/28452/commits/b8e54750a06ebd4bcbaf69236a2e8e9315565810